### PR TITLE
Multi-architecture builds to support amd64 and arm64 containers using buildx

### PR DESCRIPTION
**Pull request by austinhallett**
Use buildx action to create buildx instance. Use this instance to build containers for both amd64 and arm64.

### CODE REVIEW

**Comment by lgrosz** [comment]
Hard to parse the changes with the indentation changing

**Comment by austinhallett** [comment]
@igrosz Heard, I will undo the tab changes. Please note I am working on https://github.com/crops/yocto-dockerfiles/pull/104 in parallel to create arm-based images for the base and builder layers

**Comment by moto-timo** [comment]
We must have this working in yocto-dockerfiles FIRST because those are required to build the poky container. Also, we need to have some automated way of testing the built arm64 containers work (even if it is only the simple smoke tests). I assume this will mean qemu until GitHub provides arm64 builders to the common public. The reason this project is still viable at all is because of automation.
